xds/internal/balancer/lrs/lrs.go

```diff
@@ -152,15 +151,7 @@ type lrsStore struct {
 // NewStore creates a store for load reports.
 func NewStore(serviceName string) Store {
 	return &lrsStore{
-		node: &basepb.Node{
-			Metadata: &structpb.Struct{
-				Fields: map[string]*structpb.Value{
-					internal.GrpcHostname: {
-						Kind: &structpb.Value_StringValue{StringValue: serviceName},
-					},
-				},
-			},
-		},
```

Comment:
Will LRS be eventually moved to the xdsClient and will a nodeProto (read from the bootstrap file) be added to the request there? If not, is OK to not have a nodeProto at all in the LRS request?

Reply:
Yes, LRS will be moved to the same xdsClient.

Reply:
Should we add a TODO to say that we should add the NodeProto here once we move it to the xdsClient.

Reply:
Done